app/views/notifier/new_user.html.haml

```diff
@@ -2,8 +2,19 @@
 
 - if @password
   %p
+    - if text(".username_and_password_text").present?
+      = text(".username_and_password_text")
+      %br
```

Comment:
Can't move the `br` into the locale string unless we want to have separate strings for the html and text versions.